lib/qesapdeployment.pm

```diff
@@ -1109,16 +1109,19 @@ sub qesap_cluster_logs {
     my $inventory = qesap_get_inventory(provider => $provider);
     if (script_run("test -e $inventory") == 0)
     {
-        foreach my $host ('vmhana01', 'vmhana02') {
+        foreach my $host ('"hana[0]"', '"hana[1]"') {
```

Comment:
Is this a reference to the inventory?

Reply:
yes it is. This PR is using same strategy we used in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16744